test/core/filters/filter_fuzzer.cc

```diff
@@ -247,7 +248,10 @@ RefCountedPtr<AuthorizationEngine> LoadAuthorizationEngine(
 template <typename FuzzerChannelArgs>
 ChannelArgs LoadChannelArgs(const FuzzerChannelArgs& fuzz_args,
                             GlobalObjects* globals) {
-  ChannelArgs args = ChannelArgs().SetObject(ResourceQuota::Default());
+  ChannelArgs args =
```

Comment:
Should we just precondition the channel args here?  If these args are going to be passed to a filter, the filter will assume that they've already been preconditioned.

Reply:
Done